torchmultimodal/models/flava/flava_model.py

```diff
@@ -209,6 +209,7 @@ def flava_model_for_classification(
     classifier_activation: Callable[..., nn.Module] = nn.ReLU,
     classifier_normalization: Optional[Callable[..., nn.Module]] = None,
     loss_fn: Optional[Callable[..., Tensor]] = None,
+    pretrained_model_key: Optional[str] = "flava_full",
```

Comment:
Just wondering, why do we want to add this as a param to `flava_model_for_classification`? Feels to me like this class should not care about the pretrained weights. I like TorchVision's [approach](https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py#L730-L757) for handling this, maybe we can do something similar?

Reply:
I am just doing this for "short term uniformity" with the flava_for_pretraining. But yeah i agree, tv's approach is nicer and something we can potentially follow